resources/src/appleMain/kotlin/dev/icerock/moko/resources/desc/StringDesc.kt

```diff
@@ -4,10 +4,7 @@
 
 package dev.icerock.moko.resources.desc
 
-import platform.Foundation.NSBundle
-import platform.Foundation.NSLocale
-import platform.Foundation.currentLocale
-import platform.Foundation.localeWithLocaleIdentifier
+import platform.Foundation.*
```

Comment:
detekt will be failed with star imports

Reply:
Removed in 54753eae4645c1bf17d8a3839c4952d486f77dbe